pkg/api/common/versions.go

```diff
@@ -197,9 +198,11 @@ func getAllKubernetesWindowsSupportedVersionsMap() map[string]bool {
 
 // GetAllSupportedKubernetesVersionsWindows returns a slice of all supported Kubernetes versions on Windows
 func GetAllSupportedKubernetesVersionsWindows() []string {
-	versions := make([]string, 0, len(AllKubernetesWindowsSupportedVersions))
-	for k := range AllKubernetesWindowsSupportedVersions {
-		versions = append(versions, k)
+	var versions []string
```

Comment:
This fixes a bug,as prior it was returning all versions regardless of the bool value.

(admittedly `AllKubernetesWindowsSupportedVersions` isn't the best name for a hash table that includes explicitly not supported versions)